### PR TITLE
VZ-7965: Fetch namespaces from component registry

### PR DESCRIPTION
**Pull request by IshaGirdhar**
This PR removes the map that holds the component namespace mapping used by VZ cli tool, now tool will fetch the namespaces directly from the component registry.

### CODE REVIEW

**Review by agv426 — APPROVED** [review]

**Review by bhatpmk — APPROVED** [review]

File: pkg/constants/constants.go [hunk 1]
```diff
@@ -175,71 +173,19 @@ const VerrazzanoClusterRancherUsername = "vz-cluster-reg"
 
 // Components Names
 const (
-	OamKubernetesRuntime          = "oam-kubernetes-runtime"
```
Comment (pfmackin) [inline pkg/constants/constants.go:178]: Why did you remove several entries?

Comment (IshaGirdhar) [reply pkg/constants/constants.go:178]: These entries were used only by the intermediate map that VZ tool was using, which we are not using anymore with this PR, so removed unused constants as well, rest are used in other code in VZ

**Review by bhatpmk — APPROVED** [review]
